e2e/app/component/table.ts

```diff
@@ -99,35 +99,28 @@ export default class Table extends Container {
     return this.page.waitForXPath(cellXpath, { visible: true });
   }
 
-  async getCellByValue(rowValue: string, columnHeader: string): Promise<Cell | null> {
-    // Find the row which contains cell
-    let rowIndex = -1;
-    const rows = await this.getRows();
-    for (let i = 0; i < rows.length; i++) {
-      const tds = await rows[i].$x('./td');
-      for (let j = 0; j < tds.length; j++) {
-        const v = await getPropValue<string>(tds[j], 'innerText');
-        if (v === rowValue) {
-          rowIndex = i;
-          break;
-        }
-      }
+  async findCellByRowValue(searchColumnHeader, searchCellValue, resultColumnHeader): Promise<Cell | null> {
```

Comment:
Please add method documentation of what this is supposed to do - the semantics are not obvious from the method name.